x/ccv/provider/keeper/hooks_test.go

```diff
@@ -38,6 +46,7 @@ func TestValidatorConsensusKeyInUse(t *testing.T) {
 					newValidator.ConsumerConsAddress(),
 					anotherValidator0.ProviderConsAddress(),
 				)
+				k.SetConsumerClientId(ctx, "chainid", "clientID")
```

Comment:
NOTE: This is added because `ValidatorConsensusKeyInUse` uses `ChainToClientBytePrefix` when checking for currently registered consumer chains.